contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/ContentNodeOptions.vue

```diff
@@ -23,23 +23,15 @@
   import { mapActions, mapGetters } from 'vuex';
   import { RouterNames } from '../../constants';
   import MoveModal from '../move/MoveModal';
+  import clipboardMixin from './mixins';
   import { withChangeTracker } from 'shared/data/changes';
 
   export default {
     name: 'ContentNodeOptions',
     components: {
       MoveModal,
     },
-    props: {
-      nodeId: {
-        type: String,
-        required: true,
-      },
-      ancestorId: {
-        type: String,
-        default: null,
-      },
-    },
+    mixins: [clipboardMixin],
```

Comment:
Adding mixin which enabled removal of props and to use `contentNode` getter instead of in-file `node` getter.